app/src/main/java/com/example/tasky/feature_authentication/data/remote/TaskyAuthApi.kt

```diff
@@ -12,20 +12,17 @@ import retrofit2.http.POST
 interface TaskyAuthApi {
     @POST("/register")
     suspend fun signUp(
-        @Header("x-api-key") key: String,
         @Body registrationRequest: RegistrationRequest
     )
 
     @POST("/login")
     suspend fun signIn(
-        @Header("x-api-key") key: String,
         @Body loginRequest: LoginRequest
     ): Response<LoginResponse>
 
     @GET("/authenticate")
     suspend fun authenticate(
-        @Header("x-api-key") key: String,
-        @Header("Authorization") token: String
+        @Header("Authorization: ") token: String
```

Comment:
The colon shouldn't be needed here, Retrofit adds that. Same as for the API key you can also create an interceptor for the token :)